doctest/build.gradle

```diff
@@ -34,6 +34,22 @@ task bootstrap(type: Exec, dependsOn: ['cloneSqlCli']) {
 
 task startPrometheus(type: SpawnProcessTask) {
     doFirst {
+        def pidFile = new File(path, ".prom.pid.lock")
+        if (!pidFile.exists()) {
+            logger.quiet "No Prometheus server running!"
+        } else {
+            def pid = pidFile.text
+            def process = "kill $pid".execute()
```

Comment:
Will this work on all platforms?

Reply:
Ah, I see it's a copy of the stopPrometheus task, right?

And you can't call it from here because it generates a circular dependency?